docs/developer/getting-started.rst

```diff
@@ -111,18 +111,40 @@ To run the Python tests, run (in the ``python/`` directory):
   PYTHONPATH=$PYTHONPATH:./install python3 -m pytest
 
 Building Documentation
-----------------------
+~~~~~~~~~~~~~~~~~~~~~~
+
 - If Mantid is unavailable (e.g. on Windows) edit ``docs/conf.py`` and include ``nbsphinx_allow_errors = True``. Take care to not commit this change though.
 - run ``cmake --build . --target docs`` from your build directory.
 - This will build the documentation and put it on ``<build dir>/docs``.
-- If rebuuilding the documentation is slow it can be quicker to remove the docs build directory and start a fresh build.
+- If rebuilding the documentation is slow it can be quicker to remove the docs build directory and start a fresh build.
 
 Precommit Hooks
----------------
+~~~~~~~~~~~~~~~
 
 If you wish, you can install precommit hooks for flake8 and yapf. In the source directory run:
 
 .. code-block:: bash
 
   pre-commit install
   pre-commit run --all-files
+
+Using scipp as a C++ library
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Using Scipp as a C++ library is not recommened at this point as the API (and ABI) is not stable and documentation is sparse.
+Nonetheless, it can be used as a ``cmake`` package as follows.
+In your ``CMakeLists.txt``:
+
+.. code-block:: cmake
+
+  find_package(Scipp 0.5 REQUIRED) # replace with required version
+  target_link_libraries(mytarget PUBLIC scipp::dataset)
+
+If scipp was install using ``conda``, ``cmake`` should find it automatically.
+If you build and installed scipp from source use, e.g.,:
+
+.. code-block:: bash
+
+  cmake -DCMAKE_PREFIX_PATH=<your_scipp_install_dir>
```

Comment:
Did you consider providing a cmake find_package for scipp? I'm personally not sure it's going to help much.

Reply:
This has been added (and is improved/fixed here) by the previous PR, and it is working (see 5 lines above). `CMAKE_PREFIX_PATH` has to be used to tell `cmake` where to look. If you use `conda` it is not needed.